data_preprocessing/rd_video_image_extractor.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import multiprocessing
+from pathlib import Path
+import cv2
+
```

Comment:
Can you please add company header and a comment in PEP standard to describe the function of the file